ci/test_wheel_cugraph-dgl.sh

```diff
@@ -32,18 +32,8 @@ fi
 PYTORCH_URL="https://download.pytorch.org/whl/cu${PYTORCH_CUDA_VER}"
 DGL_URL="https://data.dgl.ai/wheels/cu${PYTORCH_CUDA_VER}/repo.html"
 
-# Starting from 2.2, PyTorch wheels depend on nvidia-nccl-cuxx>=2.19 wheel and
-# dynamically link to NCCL. RAPIDS CUDA 11 CI images have an older NCCL version that
-# might shadow the newer NCCL required by PyTorch during import (when importing
-# `cupy` before `torch`).
-if [[ "${NCCL_VERSION}" < "2.19" ]]; then
-  PYTORCH_VER="2.1.0"
-else
-  PYTORCH_VER="2.3.0"
-fi
```

Comment:
We recently bumped the NCCL floor to 2.18.1.1 in PR: https://github.com/rapidsai/cugraph/pull/4661

Reading this comment, am wondering whether we need to bump it further (namely to 2.19)

Reply:
Raising in the NCCL update issue: https://github.com/rapidsai/build-planning/issues/102#issuecomment-2375595743